app/src/main/java/com/example/b07group7project/itempreview/ItemPreviewFragment.java

```diff
@@ -0,0 +1,98 @@
+package com.example.b07group7project.itempreview;
+
+import android.os.Bundle;
+
+import androidx.annotation.NonNull;
+import androidx.fragment.app.Fragment;
+
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.Button;
+import android.widget.ImageButton;
+import android.widget.TextView;
+
+import com.example.b07group7project.R;
+
+public class ItemPreviewFragment extends Fragment {
+    private TextView cartItemQtyTextView;
+    private StoreItem currentItem;
+    private int cartItemQty = 0;
+
+    public ItemPreviewFragment() {
+        // Required empty public constructor
+    }
+
+    @Override
+    public void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+
+        // Get the item ID from arguments (sent from the previous fragment)
```

Comment:
If you're using arguments from the previous fragment, provide a way of passing those arguments